xaitk_saliency/impls/gen_descriptor_sim_sal/similarity_scoring.py

```diff
@@ -98,7 +94,7 @@ def generate(
         sal = np.clip(sal, -1, 1)
 
         # return just HxW components
```

Comment:
This comment is extraneous given we reverted back to returning NxHxW saliency maps.